js/index.js

```diff
@@ -31,7 +30,7 @@ const onCorrectAnswerHTML = {
 // Handlers for actions:
 
 const handleResponse = (questionNumber, responseTextElementId) => {
-  if (answers[questionNumber].includes(document.getElementById(responseTextElementId).value.trim())) {
+  if (answers[questionNumber].includes(document.getElementById(responseTextElementId).value.trim().toLowerCase())) {
```

Comment:
By the way, `.trim()` (which goes before `toLowerCase()`) also removes any spaces in the beginning or in the end of an input string (could be accidentally added by a user).